config.m4

```diff
@@ -342,6 +383,34 @@ if test "$PHP_MONGODB" != "no"; then
       PHP_MONGODB_ADD_BUILD_DIR([src/libmongoc/src/zlib-1.2.11/])
       AC_CONFIG_FILES([${ac_config_dir}/src/libmongoc/src/zlib-1.2.11/zconf.h])
     fi
+
+    if test "$PHP_CLIENT_SIDE_ENCRYPTION" = "yes"; then
+      dnl TODO: MONGOCRYPT-219 makes the -std argument obsolete
```

Comment:
Oh, this might be the comment I was expecting to appear above `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS`. Perhaps we should just have `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS` defined here? I don't think it's relevant to anything outside of this conditional.